PPM Never Have I Ever.py

```diff
@@ -10,19 +10,27 @@
 
 @app.route("/question")
 def question():# df):
+    choose = random.choice(Qs)
+    Qs.remove(choose)
     return '<br><br><br><br><br><br><br><br><br><br><br><center><h1>{}</h1></center>'.format(random.choice(Qs))
```

Comment:
We can use the variable choose inside the format method no need to call random.choice()